src/OrchardCore.Modules/OrchardCore.Flows/Views/ContentCard-BagPart.Edit.cshtml

```diff
@@ -5,9 +5,15 @@
 @{
     ContentItem contentItem = Model.ContentItem;
     var contentType = ContentDefinitionManager.GetTypeDefinition(contentItem.ContentType).DisplayName;
+    var collapsed = Model.Inline != true;
+    if (!ViewContext.ModelState.IsValid && ViewContext.ModelState.Keys.Any(key => key.StartsWith(Model.PrefixValue, StringComparison.OrdinalIgnoreCase)))
```

Comment:
Is `ViewContext.ModelState.IsValid` enough?

Reply:
No. because it could failed to incomplete fields outside of the bag part.